packages/stargate/src/signingstargateclient.ts

```diff
@@ -21,6 +21,7 @@ import {
   OfflineSigner,
   Registry,
 } from "@cosmjs/proto-signing";
+import { GeneratedType } from "@cosmjs/proto-signing/types/registry";
```

Comment:
This should be in the index of `@cosmjs/proto-signing` (as it is a type of the public `Registry` constructor)

Reply:
The trouble with overactive IDEs.